extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/src/main/java/org/eclipse/smarthome/binding/fsinternetradio/internal/radio/FrontierSiliconRadio.java

```diff
@@ -0,0 +1,260 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.fsinternetradio.internal.radio;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class representing a internet radio based on the frontier silicon chipset. Tested with "hama IR110" and Medion
+ * MD87180" internet radios.
+ *
+ * @author Rainer Ostendorf
+ * @author paphko
+ * @since 1.7.0
+ */
+public class FrontierSiliconRadio {
+
+    /*-
+     * [paphko]: Some snippet I found in the internet:
```

Comment:
remove this comment - it sounds as if you have copied (copyrighted!) code from other sources (which isn't the case as far as I can see)